src/resolvers/Mutation/togglePinnedPost.ts

```diff
@@ -0,0 +1,121 @@
+import {
```

Comment:
rename the function to `togglePostPin`. Everything else looks good.

Reply:
Done.